### PR TITLE
Release 0.6.0

### PR DESCRIPTION
**Pull request by fzhinkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,10 +72,19 @@ Add the library to dependencies:
 <dependency>
     <groupId>org.jetbrains.kotlinx</groupId>
     <artifactId>kotlinx-io-core-jvm</artifactId>
-    <version>0.5.4</version>
+    <version>0.6.0</version>
 </dependency>
 ```
 
+### JPMS support
```
Comment (fzhinkin) [inline README.md:79]: Is it worth explicitly mentioning it? @qwwdfsad

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## 0.6.0
+> Published 18 November 2024
+
+### Features
+- On JVM, libraries are now include Java Platform Module System (JPMS) modules [#406](https://github.com/Kotlin/kotlinx-io/pull/406)
+- Made Segment's size and capacity public [#409](https://github.com/Kotlin/kotlinx-io/pull/409)
```
Comment (shanshin) [inline CHANGELOG.md:8]: I think we should also mention changes from https://github.com/Kotlin/kotlinx-io/pull/414

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## 0.6.0
+> Published 18 November 2024
```
[the remark this answers is not in the document]
Comment (fzhinkin) [reply CHANGELOG.md:4]: I was too self-confident

**Review by shanshin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (qwwdfsad) [reply README.md:79]: I don't have an opinion on that

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## 0.6.0
+> Published 19 November 2024
+
+### Features
+- On JVM, libraries are now include Java Platform Module System (JPMS) modules [#406](https://github.com/Kotlin/kotlinx-io/pull/406)
```
Comment (lppedd) [inline CHANGELOG.md:7]: Typo: are now include > now include

**Pull request merged**